classes/widgets.php

```diff
@@ -135,7 +135,7 @@ public function widget( $args, $instance ) {
 			// Show day filters if allowed.
 			if ( $all_terms && ! empty( $show[ $taxonomy ] ) ) {
 				$php_taxonomy = str_replace( '-', '_', $taxonomy );
-				$term_search  = '<tr><td class="location_search_' . strtolower( $tax_info['singular'] ) . '_cell location_search_cell">' . apply_filters( $php_taxonomy . '-text', __( $tax_info['plural'], 'simplemap' ) ) . ':<br />';
+				$term_search  = '<tr><td class="location_search_' . strtolower( $tax_info['singular'] ) . '_cell location_search_cell">' . apply_filters( $php_taxonomy . '-text', __( $tax_info['plural'], 'SimpleMap' ) ) . ':<br />';
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 232 characters

<!-- Comment by [SideCI](https://sideci.com) -->